tests/deploys/tests/test_plotly_app.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+import pytest
+from deploy import deploy
+from playwright.sync_api import Page, expect
+
+COUNTRY = "Afghanistan"
+APP_DIR = "plotly_app"
+APP_NAME = "example_deploy_app10"
+# reqd since the app on connect takes a while to load
+PAGE_TIMEOUT = 120 * 1000
+EXPECT_TIMEOUT = 30 * 1000
+
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+app_file_path = os.path.join(os.path.dirname(current_dir), "apps", APP_DIR)
+
+
+@pytest.mark.integrationtest
+@pytest.mark.only_browser("chromium")
+@pytest.mark.parametrize("location", ["connect"])
+# TODO: Add shinyapps in the location entry
+def test_deploys(page: Page, location: str) -> None:
+    page_url = deploy(location, APP_NAME, app_file_path)
+    page.goto(page_url, timeout=PAGE_TIMEOUT)
+    expect(page.get_by_text(COUNTRY)).to_have_count(1, timeout=EXPECT_TIMEOUT)
```

Comment:
Cosmetic white space
```suggestion

    expect(page.get_by_text(COUNTRY)).to_have_count(1, timeout=EXPECT_TIMEOUT)
```